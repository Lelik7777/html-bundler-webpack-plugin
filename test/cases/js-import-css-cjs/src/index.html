<!DOCTYPE html>
<html>
<head>
  <title>Test</title>
  <link href="./style.css" rel="stylesheet">
  <link href="./inline.css?inline" rel="stylesheet">
  <script src="./app.js" defer="defer"></script>
</head>
<body>
  <div class="inline">inline.css - inlined in HTML</div>
  <div class="style">style.css - linked in HTML</div>
  <div class="app1">app1.css - imported in JS</div>
  <div class="app2">app2.css - imported in JS</div>
</body>
</html>