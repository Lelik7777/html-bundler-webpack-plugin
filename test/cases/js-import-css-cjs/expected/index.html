<!DOCTYPE html>
<html>
<head>
  <title>Test</title>
  <link href="assets/css/style.aac00a79.css" rel="stylesheet">
  <style>.inline {
  color: plum;
}
</style>
  <link href="assets/css/app.6d96bade.css" rel="stylesheet">
<script src="assets/js/app.ffec14fb.js" defer="defer"></script>
</head>
<body>
  <div class="inline">inline.css - inlined in HTML</div>
  <div class="style">style.css - linked in HTML</div>
  <div class="app1">app1.css - imported in JS</div>
  <div class="app2">app2.css - imported in JS</div>
</body>
</html>